### DISCUSSION TITLE
Add support for "comments only write access" to projects

### DISCUSSION THREAD

**Discussion by XavierGeerinck** [community/community#137645] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

I would love the ability to provide certain people access to my github project so that they can add comments (i.e. a "moderator" role) to the items in there (only comments) but not create new items / assign people / make other changes. Is this possible already?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by mkdylanw** [comment]
Anyone have a good workaround for this that they are currently doing?

ATM we are adding url to slack with a comment and mentioning @dev
